LICENSE_YQ

```diff
@@ -0,0 +1,30 @@
+Below is a copy of https://github.com/mikefarah/yq/blob/master/LICENSE
+
+This file is included in the Lima repository because `limactl yq` and
```

Comment:
Why do we need this file specifically for yq?
We do not copy the license files for other Go dependencies

Reply:
I'm not sure we need it.

I felt this is different because we are effectively bundling/distributing the `yq` command now, both for the host and the VM.

It is just compiled into our binary, which is just a packaging decision (share statically linked libraries). But otherwise the original utility is fully exposed to the end user, which is conceptually different from just using it as a library:

```console
❯ limactl yq --version
yq (https://github.com/mikefarah/yq/) version v4.47.1
```

I will remove it if you think we don't need it.

Reply:
The license file is included in https://github.com/lima-vm/lima/releases/download/v1.2.1/lima-1.2.1-go-mod-vendor.tar.gz

(If this is not enough, we have to duplicate the license files for all the deps, not just for yq)

Reply:
Good enough for me, I will remove it.